pkg/landscaper/controllers/execution/controller.go

```diff
@@ -65,7 +65,28 @@ func (c *controller) Reconcile(ctx context.Context, req reconcile.Request) (reco
 	}
 
 	if lsv1alpha1helper.IsCompletedExecutionPhase(exec.Status.Phase) {
-		logger.V(7).Info("execution item is already in completion phase")
+		op := execution.NewOperation(operation.NewOperation(logger, c.client, c.scheme, c.eventRecorder), exec, false)
+		phaseChanged, err := handleDeployItemPhaseChanges(ctx, op, logger, exec)
+		if err != nil {
+			return reconcile.Result{}, err
+		}
+		if phaseChanged {
+			if exec.Status.Phase == lsv1alpha1.ExecutionPhaseSucceeded {
```

Comment:
one indentation less
```suggestion
		if phaseChanged && exec.Status.Phase == lsv1alpha1.ExecutionPhaseSucceeded {
```